xml/entity-decl.ent

```diff
@@ -924,6 +924,40 @@ they also have a trademark policy, let's be careful here. -->
      <guimenu>Next</guimenu>.
     </para>'>
 
+<!ENTITY upgrade-with-suma-pointer '<para xmlns="http://docbook.org/ns/docbook">
+ If your machine is managed by &susemgr;, update it as described in the
+ &susemgr; documentation. The <citetitle>Client Migration</citetitle>
+ procedure is described in the <citetitle>&susemgr; Best Practices Guide</citetitle>,
+ available at <link xmlns:xlink="http://www.w3.org/1999/xlink"
+ xlink:href="https://www.suse.com/documentation/suse-manager"/>.
+ </para>'>
+
+ <!ENTITY note-upgrade-reduce-install-size '<note xmlns="http://docbook.org/ns/docbook">
+   <title xmlns="http://docbook.org/ns/docbook">Reduce Installation Size</title>
+   <para>
+    When performing the SP migration, &yast; will install all recommended
```

Comment:
```suggestion
    When performing the service pack migration, &yast; will install all recommended
```
Service pack is usually written out, the abbreviation SP only occurs 4 times in this guide (usually in tables etc.)

Reply:
Good point, I only moved over the original text from the files (to avoid duplicates), but I did not pay much attention to the wording at this point. Changed now.